server/etcdserver/api/membership/cluster.go

```diff
@@ -704,8 +704,6 @@ func clusterVersionFromBackend(lg *zap.Logger, be backend.Backend) *semver.Versi
 func downgradeInfoFromBackend(lg *zap.Logger, be backend.Backend) *DowngradeInfo {
 	dkey := backendDowngradeKey()
 	tx := be.ReadTx()
-	tx.Lock()
```

Comment:
Could you please comment why this transaction does not need to take 'lock'. 
If so, I assume we should follow the convention and call it unsafeDowngradeInfoFromBackend(...)